react/src/components/UserCredentialList.tsx

```diff
@@ -0,0 +1,491 @@
+                    }}
+                  >
+                    {record.num_queries}
+                    <Typography.Text
+                      type="secondary"
+                      style={{
+                        marginLeft: token.marginXXS,
+                        fontSize: token.fontSizeSM,
+                      }}
+                    >
+                      queries
+                    </Typography.Text>
+                  </Typography.Text>
+                </Flex>
+              );
+            },
+          },
+          {
+            key: 'control',
+            title: t('general.Control'),
+            fixed: 'right',
+            render: (record) => {
+              return (
+                <Flex gap={token.marginXS}>
+                  <Button
+                    type="link"
+                    icon={<InfoCircleOutlined />}
+                    onClick={() => {}}
+                  />
+                  <Button
+                    type="link"
+                    icon={
+                      <SettingOutlined style={{ color: token.colorInfo }} />
+                    }
+                    onClick={() => {}}
+                  />
+                  {activeType === 'inactive' && (
+                    <Button
+                      type="link"
+                      icon={<UndoOutlined style={{ color: token.colorInfo }} />}
+                      onClick={() => {
+                        commitModifyKeypair({
+                          variables: {
+                            access_key: record.access_key,
+                            props: {
+                              is_active: true,
+                            },
+                          },
+                          onCompleted: (res, errors) => {
+                            if (!res?.modify_keypair?.ok || errors) {
+                              message.error(res?.modify_keypair?.msg);
+                              return;
+                            }
+                            message.success(
+                              t('environment.SuccessfullyModified'),
+                            );
+                            startRefreshTransition(() => {
+                              updateFetchKey();
+                            });
+                          },
+                          onError: (error) => {
+                            message.error(error?.message);
+                            console.error(error);
+                          },
+                        });
+                      }}
+                    />
+                  )}
+                  <Popconfirm
+                    title={t('dialog.ask.DoYouWantToProceed')}
+                    description={t('dialog.warning.CannotBeUndone')}
+                    okButtonProps={{
+                      loading:
+                        isInFlightCommitModifyKeypair ||
+                        isInFlightCommitDeleteKeypair,
+                    }}
+                    okType="danger"
```

Comment:
If the action here is merely to change to inactive, it would be better to change the icon and message.
https://lucide.dev/icons/ban

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/XqC2uNFuj0wg8I60sMUh/5ba8703c-ecab-4198-9f30-ce75d809a9e0.png)